docs/tutorials/intro.rst

```diff
@@ -0,0 +1,47 @@
+What is Qiskit Experiments?
+===========================
+
+Qiskit Experiments is a package for running device characterization and calibration 
+experiments on top of the core functionality of Qiskit Terra.
+
+An **experiment** is a series of circuits, executed on a device, and analysis of
+of the subsequent data. Once an experiment is executed on a quantum backend, either 
+real or simulated, through a series of jobs, analysis is run automatically and results
+in the form of data and figures are generated.
+
+In addition to the experiment framework itself, Qiskit Experiments also has a rich 
+library of experiments for calibrating and characterizing qubits.
+
+What Qiskit Experiments can do
+==============================
+
+* Run characterization and calibration experiments such as quantum
+  volume and randomized benchmarking
+* Run built-in or customized experiments with all the options that Terra has
+* Specify fit series and parameters in the analysis
+* Transform the data through the data processor
+* Flexible visualization, storage, and retrieval of data
```

Comment:
In the absence of an accessible database, can we claim that we provide flexible storage?

Reply:
That's fair, I removed it.